auslib/db.py

```diff
@@ -1275,20 +1274,22 @@ def hasUrlPermission(self, username, url, method, urlOptions={}, transaction=Non
         return True
 
 
+# TODO: replace these with e-mails about rules/permissions changes?
+# https://bugzilla.mozilla.org/show_bug.cgi?id=1251338
 def getHumanModificationMonitors(systemAccounts):
     # Long lines from "what" get truncated to avoid printing out massive
     # release blobs ty the logs.
     def onInsert(table, who, what):
         if who not in systemAccounts:
-            cef_event('Human modification', CEF_ALERT, user=who, what=what, table=table.name, type='insert')
+            pass
 
     def onDelete(table, who, where):
         if who not in systemAccounts:
-            cef_event('Human modification', CEF_ALERT, user=who, where=where, table=table.name, type='delete')
+            pass
 
     def onUpdate(table, who, where, what):
         if who not in systemAccounts:
-            cef_event('Human modification', CEF_ALERT, user=who, what=what, where=where, table=table.name, type='update')
+            pass
```

Comment:
Could do, I've been treating email notification as a must have before we transition, so I didn't think it mattered. It's probably worthwhile logging them in additional to e-mail though, so I'll do this.